src/core/bypass.ts

```diff
@@ -1,5 +1,4 @@
 import { invariant } from 'outvariant'
-import { Headers } from 'headers-polyfill'
```

Comment:
Not using `Headers` polyfill in tests anymore—relying on the global API from Jest. In fact, MSW doesn't use the polyfill itself at all.